better-mouse-trap/README.md

```diff
@@ -0,0 +1,7 @@
+# Better Mouse Trap
```

Comment:
Is NF3 not getting a full README write up for hardware assembly and firmware configuration here?

Reply:
Don’t know 🤷‍♂️

For now though I figure this is better than nothing.